azure-devops/projects/pagopa-projects/service_connections.tf

```diff
@@ -173,3 +173,57 @@ resource "azurerm_key_vault_access_policy" "PROD-PAGOPA-TLS-CERT-SERVICE-CONN_kv
 
   certificate_permissions = ["Get", "Import"]
 }
+
+# DEV service connection for azure container registry 
+resource "azuredevops_serviceendpoint_azurecr" "pagopa-azurecr-dev" {
+  depends_on = [azuredevops_project.project]
+
+  project_id                = azuredevops_project.project.id
+  service_endpoint_name     = "pagopa-azurecr-dev"
+  resource_group            = "pagopa-d-aks-rg"
+  azurecr_name              = "pagopadacr"
+  azurecr_subscription_name = "DEV-PAGOPA"
+  azurecr_spn_tenantid      = module.secrets.values["PAGOPAIT-TENANTID"].value
+  azurecr_subscription_id   = module.secrets.values["PAGOPAIT-DEV-PAGOPA-SUBSCRIPTION-ID"].value
+}
+
+# UAT service connection for azure container registry 
+resource "azuredevops_serviceendpoint_azurecr" "pagopa-azurecr-uat" {
+  depends_on = [azuredevops_project.project]
+
+  project_id                = azuredevops_project.project.id
+  service_endpoint_name     = "pagopa-azurecr-uat"
+  resource_group            = "pagopa-u-aks-rg"
+  azurecr_name              = "pagopauacr"
+  azurecr_subscription_name = "UAT-PAGOPA"
+  azurecr_spn_tenantid      = module.secrets.values["PAGOPAIT-TENANTID"].value
+  azurecr_subscription_id   = module.secrets.values["PAGOPAIT-UAT-PAGOPA-SUBSCRIPTION-ID"].value
+}
+
+# PROD service connection for azure container registry 
+resource "azuredevops_serviceendpoint_azurecr" "pagopa-azurecr-prod" {
+  depends_on = [azuredevops_project.project]
+
+  project_id                = azuredevops_project.project.id
+  service_endpoint_name     = "pagopa-azurecr-prod"
+  resource_group            = "pagopa-p-aks-rg"
+  azurecr_name              = "pagopapacr"
+  azurecr_subscription_name = "PROD-PAGOPA"
+  azurecr_spn_tenantid      = module.secrets.values["PAGOPAIT-TENANTID"].value
+  azurecr_subscription_id   = module.secrets.values["PAGOPAIT-PROD-PAGOPA-SUBSCRIPTION-ID"].value
+}
+
+# SIA service connection (read-write)
```

Comment:
docker registry are standard.. we create it as other and this configs:
url: docker-registry-default.ocp-tst-npaspc.sia.eu
user: serviceaccount
pwd: module.secrets.values["DEV-PAGOPA-SIA-DOCKER-REGISTRY-PWD"].value